apps/omg_watcher_rpc/lib/release_tasks/set_endpoint.ex

```diff
@@ -0,0 +1,59 @@
+# Copyright 2019-2019 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.WatcherRPC.ReleaseTasks.SetEndpoint do
+  @moduledoc false
+  use Distillery.Releases.Config.Provider
+  require Logger
+  @app :omg_watcher_rpc
+
+  @impl Provider
+  def init(_args) do
+    _ = Application.ensure_all_started(:logger)
+    config = Application.get_env(@app, OMG.WatcherRPC.Web.Endpoint)
+    config = Keyword.put(config, :disabled?, get_port())
```

Comment:
I only noticed this in this file, but maybe worth to check other ones if I'm right:

Putting `get_port` to `:disabled?` is suspicious